access/src/main/resources/holdingsitems/migration/V11__BranchIdBibliographicRecordIdIndex.sql

```diff
@@ -0,0 +1,15 @@
+DO
```

Comment:
Hvad er fordelen ved denne metode fremfor bare

CREATE INDEX ... IF NOT EXIST ON ...

Reply:
Det kan godt være at det kun var gamle pg'er, men jeg har set den eksplodere hvis kolonnerne ikke var nævnt i samme rækkefølge. Dét burde ikke ske, men.... der er en grund til at jeg ved det.

Reply:
Man kunne selvfølgelig argumentere for at det skulle fejle...

Reply:
Jeg har lige checket... den fejler ikke længere. Jeg laver om.